sound/soc/intel/boards/sof_maxim_common.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Copyright(c) 2020 Intel Corporation.
+ */
+
+/*
+ * This file defines data structures used in Machine Driver for Intel
+ * platforms with Maxim Codecs.
+ */
+#ifndef __SOF_MAXIM_COMMON_H
+#define __SOF_MAXIM_COMMON_H
+
+#define MAX_98373_CODEC_DAI	"max98373-aif1"
+#define MAX_98373_DEV0_NAME	"i2c-MX98373:00"
+#define MAX_98373_DEV1_NAME	"i2c-MX98373:01"
+
+extern const struct snd_soc_dapm_route max_98373_dapm_routes[];
+extern struct snd_soc_codec_conf max_98373_codec_conf[];
```

Comment:
these two can be hidden as static, and since you use ARRAY_SIZE they should definitively be static

Reply:
Thank you. Removed them and added static for those in c file. Updated patches now.